source/nodejs/adaptivecards/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "adaptivecards",
-	"version": "1.2.0",
+	"version": "1.3.0-alpha1",
```

Comment:
>: "1.3.0-alpha1", [](start = 10, length = 17)

am guessing this change is by design and you made it?

Reply:
I don't see a reason why it shouldn't be 1.3.0-alpha1? It can always become anything else we want if necessary.

Reply:
I am not sure why this is getting changed esp since this ends pu in a totatlly separate package right? This shouldnt be touched wiht this change yes?

---
In reply to: [297856584](https://github.com/microsoft/AdaptiveCards/pull/2995#discussion_r297856584) [](ancestors = 297856584)

Reply:
Yes it should. We have shipped 1.2, what lives in master needs to have a newer version.